assets/js/src/core/modules/notifications/send-notification/send-notification-modal.tsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * This source file is available under the terms of the
+ * Pimcore Open Core License (POCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (https://www.pimcore.com)
+ *  @license    Pimcore Open Core License (POCL)
+ */
+
+import { Form } from '@Pimcore/components/form/form'
+import { WindowModal } from '@Pimcore/components/modal/window-modal/window-modal'
+import { ManyToOneRelation } from '@Pimcore/modules/element/dynamic-types/definitions/objects/data-related/components/many-to-one-relation/many-to-one-relation'
+import { FieldWidthProvider } from '@Pimcore/modules/element/dynamic-types/definitions/objects/data-related/providers/field-width/field-width-provider'
+import { Input } from 'antd'
```

Comment:
Same here, can we use ours instead?

Reply:
done with https://github.com/pimcore/studio-ui-bundle/pull/1515/commits/3e7064f5c8cbfede28bb96a7452236c353d895f3